install_config/aggregate_logging.adoc

```diff
@@ -1725,6 +1725,10 @@ aggregator, in addition to the default Elasticsearch, using the `secure-forward`
 plug-in. From there, you can further process log records after the locally
 hosted Fluentd has processed them.
 
+[IMPORTANT]
+====
+You cannot configure the `secure_foward` plug-in with a client certificate. Authentication can be run via SSL protocol and `shared_key` but require the `Fluentd` to be configured with the `secure_foward` input plug-in.
```

Comment:
The ISG suggests avoiding `via` I think `through` or `by` would work here.

Reply:
I am not sure if this applies here but the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#ssl-tls) states to us SSL/TLS instead of SSL only.

Reply:
@anpingli, what's your opinion of SSL/TLS instead of just SSL?

Reply:
SSL/TLS is better than SSL.